 <launch>
   <arg name="goal"/>
   <node pkg="robot_pose_publisher" type="robot_pose_publisher" name="robot_pose_publisher"/>
   <node pkg="jackal_affordance" type="namo_planner" name="hsr_namo_planner" args="$(arg goal)" output="screen">
     <rosparam>
       trajectory_topic: /base_local_path_ref
       dynamic_map_topic: /dynamic_obstacle_map_ref
       movebase_cancel_topic: /move_base/move/cancel
       fixed_frame: map
     </rosparam>
   </node>
 </launch>
